registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/utils/Utilities.java

```diff
@@ -259,6 +261,11 @@ public int getApplicantAge(String id, String process, ProviderStageName stageNam
 		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
 				id, "Utilities::getApplicantAge()::entry");
 
+		// return age as -1 in case of LOST packet
+		if (RegistrationType.LOST.name().equalsIgnoreCase(process)) {
+			return -1;
+		}
```

Comment:
We need a standard change, tommorow there might be another process which might have not contain the age or UIN. So lets not compare with a process name.

Reply:
checking the age and uin fields instead of process name...